cafebabel/templates/articles/tags/edit.html

```diff
@@ -0,0 +1,41 @@
+{% extends "base.html" %}
+
```

Comment:
We may want to add some basic metas (title only?)

Reply:
We don't need metas on edit pages.

Reply:
I know we don't need SEO but the title might be an indication for history and title bar.
Not big priority though I guess.

Reply:
Same apply to all edit pages, transversal issue?

Reply:
I suppose it is transverse, but as we are working in // on the meta data it may be why it pops out now.
Not a blocker though.

Reply:
There is already way too much things to fix within that sprint.